### PR TITLE
Upstreaming fix readme

### PR DESCRIPTION
**Pull request by ioannisg**

### CODE REVIEW

**Comment by ioannisg** [comment]
@oyvindronningstad FYI

File: platform/ext/target/nordic_nrf/nrf5340pdk_nrf5340_cpuapp/README.rst [hunk 1]
```diff
@@ -37,12 +37,9 @@ following commands:
 .. code:: bash
 
     $ mkdir build && cd build
-    $ cmake -G"Unix Makefiles" \
-            -DPROJ_CONFIG=`readlink -f ../configs/ConfigDefault.cmake` \
-            -DTARGET_PLATFORM=nrf5340pdk_nrf5340_cpuapp \
-            -DCMAKE_BUILD_TYPE=Debug \
-            -DBL2=False \
-            -DCOMPILER=GNUARM \
+    $ mkdir build && cd build
```
Comment (ioannisg) [inline platform/ext/target/nordic_nrf/nrf5340pdk_nrf5340_cpuapp/README.rst:40]: Now these sample outputs are similar to the NXP board.

**Comment by oyvindronningstad** [comment]
cc @anangl

**Review by oyvindronningstad — APPROVED** [review]

File: platform/ext/target/nordic_nrf/nrf5340pdk_nrf5340_cpuapp/README.rst (hunk 1 above)
Comment (anangl) [reply platform/ext/target/nordic_nrf/nrf5340pdk_nrf5340_cpuapp/README.rst:40]: But why is that line present twice?

Comment (oyvindronningstad) [reply platform/ext/target/nordic_nrf/nrf5340pdk_nrf5340_cpuapp/README.rst:40]: I'll fix it in my next PR

Comment (ioannisg) [reply platform/ext/target/nordic_nrf/nrf5340pdk_nrf5340_cpuapp/README.rst:40]: ups sorry @anangl  this came from copy-paste.
ashamed now

**Pull request merged**